selenium/base/base_1/Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM ubuntu:16.04
 
 RUN \
     apt-get update && \
-    apt-get install -y apt-transport-https ca-certificates && \
+    apt-get install -y apt-transport-https ca-certificates tzdata && \
```

Comment:
No, this file will not change.

Reply:
This package is required if `base_1` is intended to be installed:

```
RUN \
    echo ttf-mscorefonts-installer msttcorefonts/accepted-mscorefonts-eula select true | debconf-set-selections && \
    echo 'UTC' | tee /etc/timezone && \
    dpkg-reconfigure -f noninteractive tzdata && \
    echo "gtk-cursor-blink=0" > /root/.gtkrc-2.0
```
In any case if you prefer I can drop it, but `base_1` would useless again.